core/tests/protractor_utils/LibraryPage.js

```diff
@@ -79,6 +82,14 @@ var LibraryPage = function() {
     return waitFor.pageToFullyLoad();
   };
 
+  this.addToPlaylist = function() {
```

Comment:
What will happen if there are multiple explorations in the library, will it add all of them to the library? I was expecting this method to accept a parameter which will indicate the exploration which needed to be added in the play later list.

Reply:
Done!